resources/warnings.json

```
@@ -7124,7 +7124,7 @@
     "versions": [
       {
         "lastVersion": "1.1",
-        "pattern": ".*"
+        "pattern": "1[.][01][.].+"
```

Since its a relatively small finite list, can i just cheat and list them out 
```suggestion
       "pattern": "^(0[.]1|0[.]10|0[.]11|0[.]12|0[.]13|0[.]14|0[.]14[.]1|0[.]15|0[.]16|0[.]17|0[.]18|0[.]19|0[.]19[.]1|0[.]19[.]2|0[.]2|0[.]3|0[.]4|0[.]4[.]1|0[.]5|0[.]6|0[.]6[.]1|0[.]7|0[.]8|0[.]9|1[.]0|1[.]1|1[.]1[.]1)$",
```

---

```suggestion
        "pattern": "(0|1[.][01])(|[.-].+)"
```